exchanges/bybit/bybit_wrapper.go

```diff
@@ -1640,519 +1300,341 @@ func (by *Bybit) GetOrderHistory(ctx context.Context, req *order.MultiOrderReque
+					Open:   klineItems[i].Open,
+					High:   klineItems[i].High,
+					Low:    klineItems[i].Low,
+					Close:  klineItems[i].Close,
+					Volume: klineItems[i].TradeVolume,
 				})
 			}
-		default:
-			return nil, fmt.Errorf("%s %w", req.Asset, asset.ErrNotSupported)
 		}
+		return req.ProcessResponse(timeSeries)
+	default:
+		return nil, fmt.Errorf("%s %w", a, asset.ErrNotSupported)
 	}
-	return req.ProcessResponse(timeSeries)
 }
 
 // GetServerTime returns the current exchange server time.
-func (by *Bybit) GetServerTime(ctx context.Context, a asset.Item) (time.Time, error) {
-	switch a {
-	case asset.Spot:
-		info, err := by.GetSpotServerTime(ctx)
-		if err != nil {
-			return time.Time{}, err
-		}
-		return info, nil
-	case asset.CoinMarginedFutures, asset.USDTMarginedFutures, asset.Futures, asset.USDCMarginedFutures:
-		info, err := by.GetFuturesServerTime(ctx)
-		if err != nil {
-			return time.Time{}, err
-		}
-		return info, nil
+func (by *Bybit) GetServerTime(ctx context.Context, _ asset.Item) (time.Time, error) {
+	info, err := by.GetBybitServerTime(ctx)
+	if err != nil {
+		return time.Time{}, err
 	}
-	return time.Time{}, fmt.Errorf("%s %w", a, asset.ErrNotSupported)
+	return info.TimeNano.Time(), err
 }
 
 // UpdateOrderExecutionLimits sets exchange executions for a required asset type
 func (by *Bybit) UpdateOrderExecutionLimits(ctx context.Context, a asset.Item) error {
-	var limits []order.MinMaxLevel
+	var err error
+	var instrumentsInfo *InstrumentsInfo
 	switch a {
-	case asset.Spot:
-		pairsData, err := by.GetAllSpotPairs(ctx)
+	case asset.Spot, asset.USDTMarginedFutures, asset.USDCMarginedFutures, asset.CoinMarginedFutures:
+		instrumentsInfo, err = by.GetInstrumentInfo(ctx, getCategoryName(a), "", "", "", "", 400)
 		if err != nil {
 			return err
 		}
-
-		limits = make([]order.MinMaxLevel, 0, len(pairsData))
-		for x := range pairsData {
-			var pair currency.Pair
-			var enabled bool
-			pair, enabled, err = by.MatchSymbolCheckEnabled(pairsData[x].Name, a, false)
-			if err != nil {
-				log.Warnf(log.ExchangeSys, "%s unable to load limits for %v, pair data missing", by.Name, pairsData[x].Name)
-				continue
-			}
-			if !enabled {
-				continue
-			}
-
-			limits = append(limits, order.MinMaxLevel{
-				Asset:                   a,
-				Pair:                    pair,
-				AmountStepIncrementSize: pairsData[x].BasePrecision.Float64(),
-				QuoteStepIncrementSize:  pairsData[x].QuotePrecision.Float64(),
-				MinimumBaseAmount:       pairsData[x].MinTradeQuantity.Float64(),
-				MaximumBaseAmount:       pairsData[x].MaxTradeQuantity.Float64(),
-				MinimumQuoteAmount:      pairsData[x].MinTradeAmount.Float64(),
-				MaximumQuoteAmount:      pairsData[x].MaxTradeAmount.Float64(),
-				PriceStepIncrementSize:  pairsData[x].MinPricePrecision.Float64(),
-			})
+	case asset.Options:
+		instrumentsInfo, err = by.GetInstrumentInfo(ctx, getCategoryName(a), "", "", "BTC", "", 400)
+		if err != nil {
+			return err
```

Comment:
Now that we setup options to be for both BTC and ETH, you need to do the following:
```go
		var ethInstruments *InstrumentsInfo
		instrumentsInfo, err = by.GetInstrumentInfo(ctx, getCategoryName(a), "", "", "BTC", "", 400)
		if err != nil {
			return err
		}
		ethInstruments, err = by.GetInstrumentInfo(ctx, getCategoryName(a), "", "", "ETH", "", 400)
		if err != nil {
			return err
		}
		instrumentsInfo.List = append(instrumentsInfo.List, ethInstruments.List...)
```